spec/acceptance/disable_new_tasking_system_spec.rb

```diff
@@ -0,0 +1,129 @@
+require 'spec_helper_acceptance'
+
+describe 'initial configuration with newer postgresql tasking system' do
+  certdir = '/etc/pulpcore-certs'
+
+  it_behaves_like 'an idempotent resource' do
+    let(:manifest) do
+      <<-PUPPET
+      class { 'pulpcore':
+        worker_count          => 1,
+        use_rq_tasking_system => false,
+      }
+      PUPPET
+    end
+  end
+
+  describe service('httpd') do
+    it { is_expected.to be_enabled }
+    it { is_expected.to be_running }
+  end
+
+  describe service('pulpcore-api') do
+    it { is_expected.to be_enabled }
+    it { is_expected.to be_running }
+  end
+
+  describe service('pulpcore-content') do
+    it { is_expected.to be_enabled }
+    it { is_expected.to be_running }
+  end
+
+  describe service('pulpcore-resource-manager') do
```

Comment:
I thought this service went away with the new system

Reply:
the new entrypoint `pulpcore-worker --resource-manager` points to the old resource manager when the app is configured for it, else it is basically a null command

Reply:
I've been waiting to see how this behaves once the new tasking system is available, to see if tests need updating

Reply:
I see. I think I am OK with that behavior for the first pass of this. We should clean that up later so the service is gone in the new tasking system.

Reply:
@mdellweg @bmbouter can you enlighten us?

Reply:
This service is no longer needed at all with the new tasking system. Starting it will not error and it will just sleep. At some point starting it will yield an error.

Reply:
Thanks! Given that we've declared the module 3.14 only, we can ensure it's present & running if RQ is used and absent & stopped when new tasking is used. This will save some resources but more importantly, it's easier to understand for users.

Reply:
> Thanks! Given that we've declared the module 3.14 only, we can ensure it's present & running if RQ is used and absent & stopped when new tasking is used. This will save some resources but more importantly, it's easier to understand for users.

By removing the service under those circumstances, we'd be creating an issue in `foreman-maintain service` and in any other scripts that check service status. I'm appreciative of the grace period we've been given by the current transitionary design and I think it would be beneficial to proceed carefully here.

Reply:
You can then just ensure the service is present but stopped. I think it should deal with disabled services now. Also, foreman-maintain shouldn't hold us back and if it can't deal with it, we should fix it.

Reply:
I agree foreman-maintain should handle it, the question that I think is a good one being raised by @wbclark is the radius of affects for this first change. So we have two options:

 Option 1:
   * Leave resource manager alone
   * Update foreman-maintain and any other code
   * Add another patch that better manages resource-manager
  
Option 2:
 * Handle resource-manager from the start
 * Fix up foreman-maintain and other areas
 
I do agree that resource-manager being present and running presents the biggest over head to debugging as it may lead folks down incorrect paths when investigating. That is something I do think is important to avoid where we can given the newness of this.

Reply:
It's been mentioned that there is a resource overhead for having the extraneous service, but my understanding is that it should be quite negligible.

It's been mentioned as well that there is an overhead to debugging as the presence of the resource manager could lead people down the wrong path in troubleshooting. I would argue that the _absence_ of the resource manager could have the very same effect.

For me, it comes down to two things: 1. the radius of effects, as @ehelms called it, and 2. what I read to be the Pulp team's design intent in implementing an idle "resource manager" under the PostgreSQL tasking system (I infer they made this decision for essentially the same reasons as the 1st point, but applied to an even wider set of users).

So my strong preference, given the context in which we are transitioning to pulpcore 3.14, is to minimize the risk and take advantage of the opportunity to easily switch between tasking systems without having to manage a different set of services.

Reply:
> Update foreman-maintain and any other code

It's the "and any other code" part here that really concerns me. Maybe that's no code at all, maybe that's end user scripts and in house monitoring tools that we aren't responsible for, or maybe there are other things which we're responsible for which could break. I'm not 100% confident about it, and that's the reason I'd prefer to defer the change.

Reply:
I lean towards saying: look, we changed the tasking system, if your local custom code broke by that, fix it now, please.

Pretending the manager is still there would just break it later and for now give the users the false feeling of their old code still working (it is not, it is looking at a dummy).

I *think* f-maintain should be able to handle this situation today, and if not we can fix it. Any other code we shouldn't care about breaking, not at a point like this were we are actively replacing big parts of the stack.

Reply:
I lean the same way as @evgeni. We're going to have to deal with it at some point anyway and as a user I would be very unhappy with a service that does nothing. I also think that the customer support aspect should weigh heavy. In theory it can fail and does generate output so it means people will check it. That's only wasting time.